cassandra/cassandra-docker.sh

```diff
@@ -286,7 +286,7 @@ if [ -n "$KEYSTORE_PASSWORD_FILE" ]; then
 fi
 if [ ! -n "$KEYSTORE_FILE" ]; then
     KEYSTORE_FILE="${KEYSTORE_DIR}/.keystore"
-    KEYSTORE_PASSWORD=${KEYSTORE_PASSWORD:-$(head /dev/urandom -c 512 | tr -dc A-Z-a-z-0-9 | head -c 17)}
+    KEYSTORE_PASSWORD=${KEYSTORE_PASSWORD:-$(openssl rand -base64 512 | tr -dc A-Z-a-z-0-9 | head -c 17)}
```

Comment:
I don't think there's a limit for the Keystore's password length, so, it can be as long as we deemed necessary. To be honest, I just copied this routine from another script I had, for a completely different password. In fact, I think we are focusing too much on this not-so-important password: if an attacker gets access to the keystore/truststore, chances are high that the attacker also got access to a configuration file with this password in clear text.

I'm OK to change this, but I'd prefer to have it done in a way that can be reused elsewhere as well. If there's a common way it's done for OpenShift, let me know and I'll just use it.